wcfsetup/install/files/js/WoltLabSuite/Core/Ui/Reaction/Handler.js

```diff
@@ -372,6 +383,10 @@ define(
 				if (this._popoverCurrentObjectId !== 0) {
 					this._getPopover().classList.remove('active');
 					
+					elBySelAll('.reactionTypeButton[data-is-usable="0"]', this._getPopover(), function (element) {
```

Comment:
Small optimization (saves one closure).
```suggestion
					elBySelAll('.reactionTypeButton[data-is-usable="0"]', this._getPopover(), elHide);
```